client/my-sites/marketing/buttons/options.jsx

```diff
@@ -195,43 +196,72 @@ class SharingButtonsOptions extends Component {
 		);
 	}
 
-	render() {
-		const { initialized, saving, settings, siteId, translate } = this.props;
+	getSharingShowOptionsElement = () => {
+		const { initialized, isSharingShowAllowed, settings, translate } = this.props;
+
+		if ( ! isSharingShowAllowed ) {
+			return;
+		}
+
 		const changeSharingPostTypes = partial( this.handleMultiCheckboxChange, 'sharing_show' );
+		return (
+			<fieldset className="sharing-buttons__fieldset">
+				<legend className="sharing-buttons__fieldset-heading">
+					{ translate( 'Show sharing buttons on', {
+						context: 'Sharing options: Header',
+						comment:
+							'Possible values are: "Front page, Archive Pages, and Search Results", "Posts", "Pages", "Media"',
+					} ) }
+				</legend>
+				<MultiCheckbox
+					name="sharing_show"
+					options={ this.getDisplayOptions() }
+					checked={ settings.sharing_show }
+					onChange={ changeSharingPostTypes }
+					disabled={ ! initialized }
+				/>
+			</fieldset>
+		);
+	};
 
+	getWpAdminBanner = () => {
+		const { isSharingShowAllowed, siteAdminUrl, translate } = this.props;
+		if ( isSharingShowAllowed ) {
+			return;
+		}
 		return (
-			<div className="sharing-buttons__panel">
-				{ siteId && <QueryPostTypes siteId={ siteId } /> }
-				<h4>{ translate( 'Options' ) }</h4>
-				<div className="sharing-buttons__fieldset-group">
-					<fieldset className="sharing-buttons__fieldset">
-						<legend className="sharing-buttons__fieldset-heading">
-							{ translate( 'Show sharing buttons on', {
-								context: 'Sharing options: Header',
-								comment:
-									'Possible values are: "Front page, Archive Pages, and Search Results", "Posts", "Pages", "Media"',
-							} ) }
-						</legend>
-						<MultiCheckbox
-							name="sharing_show"
-							options={ this.getDisplayOptions() }
-							checked={ settings.sharing_show }
-							onChange={ changeSharingPostTypes }
-							disabled={ ! initialized }
-						/>
-					</fieldset>
-					{ this.getCommentLikesOptionElement() }
-					{ this.getTwitterViaOptionElement() }
-				</div>
+			<Banner
+				className="sharing-buttons__banner"
+				href={ `${ siteAdminUrl }options-general.php?page=sharing` }
+				title={ translate( 'Visit WP Admin for more sharing buttons options.' ) }
```

Comment:
Small question -- I don't actually know the answer to this, but is there a standard way of referring to a site's wp-admin to users? Didn't we used to call it the dashboard or something?

Reply:
I had the same doubt and just ran a simple search across the repo, and it's mentioned as "WP Admin" in several occasions, for example, in the link at the bottom of the sidebar.